### PR TITLE
[BUG] Update mobile and passport regex

### PR DESCRIPTION
**Pull request by lyeyixian**

### CODE REVIEW

File: src/main/java/seedu/address/logic/command/add/AddFriendCommand.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_MOBILE + "91234567 "
-            + PREFIX_PASSPORT + "p12345678 ";
+            + PREFIX_PASSPORT + "P1234567 ";
```
Comment (underthehai) [inline src/main/java/seedu/address/logic/command/add/AddFriendCommand.java:26]: I think passport should end with a character

**Review by underthehai — APPROVED** [review]
LGTM, just the minor part with passport in addfriendcommand

File: src/main/java/seedu/address/model/util/SampleWanderlustDataUtil.java [hunk 2]
```diff
@@ -23,20 +23,20 @@
 
     public static Friend[] getSampleFriends1() {
         return new Friend[] {
-            new Friend(new Name("Alex Yeoh"), new Passport("M1234567"), new Mobile("87438807")),
-            new Friend(new Name("Bernice Yu"), new Passport("E1234567"), new Mobile("99272758")),
-            new Friend(new Name("Charlotte Oliveiro"), new Passport("C1234567"), new Mobile("93210283")),
-            new Friend(new Name("David Li"), new Passport("D1234567"), new Mobile("99103182"))
+            new Friend(new Name("Alex Yeoh"), new Passport("S1234567R"), new Mobile("87438807")),
```
Comment (jiaweiteo) [inline src/main/java/seedu/address/model/util/SampleWanderlustDataUtil.java:26]: Prefix of SG passport is E, not S

**Review by jiaweiteo — APPROVED** [review]
LGTM, need to change prefix

File: src/main/java/seedu/address/model/util/SampleWanderlustDataUtil.java (hunk 2 above)
Comment (lyeyixian) [reply src/main/java/seedu/address/model/util/SampleWanderlustDataUtil.java:26]: huh? It's not [STFG] ?

**Pull request merged**